resources/references/adr/checkout/2021-10-01-payment-flow.md

```diff
@@ -21,15 +21,15 @@ The synchronous payment is intended to execute a payment immediately after the o
 
 The following diagram shows a happy case sequence of a synchronous payment handling. The error handling is described [here](#after-order-payment-error-case) 
 
-![Synchronous Payment](../assets/payment-flow/synchronous-payment.png)
+![Synchronous Payment](../../../../.gitbook/assets/adr/payment-flow/synchronous-payment.png)
```

Comment:
This is correct, but it should have been generated like this by https://github.com/shopware/docs/blob/master/.github/scripts/update-adrs.sh

Reply:
Technically, images have incorrect links in `shopware/platform` - [[2x here]](https://github.com/shopware/platform/blob/trunk/adr/checkout/2021-10-01-payment-flow.md), [[1x here]](https://github.com/shopware/platform/blob/trunk/adr/performance/2021-11-09-increment-pattern.md). Then I can cover both cases (`../assets/` and `./assets/`) correctly in the `update-adrs.php`.